src/js/bodyparser.js

```diff
@@ -0,0 +1,40 @@
+// Module to add body parsing to express.
+"use strict";
+var namespace  = "gpii.express.middleware.bodyparser";
+var fluid      = fluid || require('infusion');
+var gpii       = fluid.registerNamespace("gpii");
+var bodyParser = fluid.registerNamespace(namespace);
+var bp         = require("body-parser");
+
+bodyParser.json = function(that, req, res, next) {
+    if (!that.privateJson) {
+        that.privateJson       = bp.json();
```

Comment:
Some of this is gone with the refactor.  However, this was an intentional, and an extension of our discussion about using whitespacing to line up json values independently of the key length.